lib/components/home/artwork_rails/artwork_rail.js

```diff
@@ -35,44 +34,38 @@ class ArtworkRail extends React.Component {
     this.setState({ expanded: true })
   }
 
-  renderExpansionButton() {
-    if (!this.state.expanded) {
-      return (
-        <TouchableHighlight style={styles.expansionButton} onPress={this.expand} underlayColor={'white'}>
-            <Image style={{height: 8, width: 15, alignSelf: 'center'}} source={require('../../../../images/chevron.png')} />
-        </TouchableHighlight>
-      )
-    }
-    return null
-  }
-
-  renderViewAllButton() {
-    if (this.state.expanded) {
-      return (
-        <TouchableHighlight style={styles.viewAllButton} onPress={this.handleViewAll} underlayColor={'gray'}>
-          <Text style={styles.viewAllText}>VIEW ALL</Text>
-        </TouchableHighlight>
-      )
-    }
-    return null
+  hasAdditionalContent() {
+    const additionalContentRails = [
+      'followed_artists',
+      'saved_works',
+      'live_auctions',
+      'current_fairs',
+      'genes',
+      'generic_gene',
+    ]
```

Comment:
I prefer such constants to move to the top of the file so that they stand out more.